Website/index.html

```diff
@@ -0,0 +1,118 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Funcky</title>
+    <link href="./fonts.css" rel="stylesheet" />
+    <link href="./style.css" rel="stylesheet" />
+</head>
+<body>
+    <nav class="navigation-block" role="navigation">
+        <ul role="presentation" class="_container">
+            <li role="presentation">
+                <a href="./book">Documentation</a>
+            </li>
+            <li role="presentation">
+                <a href="https://github.com/polyadic/funcky">GitHub</a>
+            </li>
+            <li role="presentation">
+                <a href="https://www.nuget.org/profiles/polyadic">NuGet</a>
+            </li>
+        </ul>
+    </nav>
+    <main>
+        <header>
+            <div class="_container">
+                <h1>Funcky</h1>
+                <p class="subtitle">
+                    Funcky is a library that empowers C# developers
+                    to write code using <strong>functional programming</strong> paradigms.
+                </p>
+            </div>
+        </header>
+        <section class="section-block">
+            <div class="_container">
+                <h2>Why Funcky?</h2>
+                <p>
+                    Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+                    Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
+                    Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.
+                    Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
```

Comment:
I haven't written a text here, maybe @FreeApophis has an idea what's important to say :)